packages/node_modules/@ciscospark/widget-recents/src/components/RecentsHeader.js

```diff
@@ -38,20 +41,29 @@ function RecentsHeader(props) {
     hideBottomBorder,
     onAddClick,
     onFilterChange,
-    onProfileClick
+    onProfileClick,
+    onSignOutClick
   } = props;
 
+  const content = <PopoverContent currentUser={currentUser} onSignOutClick={onSignOutClick} />;
+
   return (
     <div className={classNames('ciscospark-recents-header', styles.recentsHeader, 'cui--dark', hideBottomBorder ? '' : styles.bottomBorder)}>
       <div className={classNames('ciscospark-recents-header-left', styles.headerSideItem)}>
-        {
-          enableUserProfile &&
+        { enableUserProfile &&
```

Comment:
We need a config for enabling the popover. Just having the user profile enabled shouldn't show the popover.